### PR TITLE
anthropic model (Sonet tested) and a asset file added

### PR DESCRIPTION
**Pull request by firojalam**

### CODE REVIEW

File: assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+from llmebench.datasets import ThatiARDataset
+from llmebench.models import AnthropicModel
+from llmebench.tasks import SubjectivityTask
+
+
+def metadata():
+    return {
+        "author": "Arabic Language Technologies, QCRI, HBKU",
+        "model": "claude-3-5-sonnet-20240620",
+        "description": "Anthropic model",
```
Comment (MaramHasanain) [inline assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py:12]: I think a more informative description is needed?

File: assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py
Comment (MaramHasanain) [inline assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py:1]: I think the asset name isn't consistent with what we had before, e.g., "Adult_GPT4_ZeroShot"

File: assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        },
+    }
+
+
+def prompt(input_sample):
+
+    prompt = f"""
+        Classify the following Arabic 'sentence' as subjective or objective. Provide only the label. 
+        Provide your response in the following JSON format: {{"label": "your label"}}. 
+        Please provide JSON output only. No additional text.
```
Comment (MaramHasanain) [inline assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py:36]: Why json? I think adding this part made the prompt too long.

File: assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        {
+            "role": "user",
+            "content": prompt,
+        },
+    ]
+
+
+def post_process(response):
+    data = response["content"][0]["text"].lower()
+    data = json.loads(data)
```
Comment (MaramHasanain) [inline assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py:50]: From my experience, loading model output as json directly might break,

File: assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py
Comment (firojalam) [reply assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py:1]: this is one of the previous asset name for subjectivity, same convention here, right ? 

CT23Subjectivity_Jais13b_ZeroShot

File: assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py (hunk 2 above)
Comment (firojalam) [reply assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py:36]: For original implementation, we also asked to provide explanation, score etc. For which json structure is important to get them right.

File: assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py (hunk 3 above)
Comment (firojalam) [reply assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py:50]: See above.

File: assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py
Comment (MaramHasanain) [reply assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py:1]: yeah but your asset name has "Sonet" in it, not sure why when the mode is Claude

**Pull request merged**